frontend/src/state/backendApi.ts

```diff
@@ -1058,7 +1061,7 @@ const apiStore = {
     },
 
 
-    async refreshConnectClusters(force?: boolean): Promise<void> {
+    async refreshConnectClusters(force?: boolean): Promise<void | KafkaConnectors> {
```

Comment:
Isn't this endpoint supposed to *always* return ``KafkaConnectors`? 
If yes, then there should be no `| void` here.

If an error happens (can be that connection didn't work, or can be that the backend returned an error response), that is ok, then an error/exception will be thrown.

Generally there should *never* be a `T | void` type in any promise result

Reply:
I did need the value coming from this function, but I didn't want to rewrite the cachedResponsed logic, with the current implementation, is a bit hard and patchy to workaround the `T | void` type declaration, maybe we can keep this for a future change